extensions/wikia/Wall/WallHooksHelper.class.php

```diff
@@ -686,8 +686,33 @@ static public function onArticleDelete( $article, &$user, &$reason, &$error ) {
 		return true;
 	}
 
-	static public function onRecentChangeSave( $recentChange ) {
-		wfProfileIn( __METHOD__ );
+	/**
+	 * Adjusting Recent Changes for Wall
+	 * Change RC db key to affect Watchlist behavior
+	 * @author TK-999 <https://github.com/TK-999>
+	 * @param RecentChange $rc
+	 * @return bool true because it is a hook
```

Comment:
I'm not sure we have any guidelines for this so I would suggest being consistent within the file and/or changeset.